.circleci/ci-oldest-reqs.txt

```diff
@@ -2,3 +2,4 @@ cloudpickle==1.1.1
 deprecation==2.0.0
 jinja2==2.10
 tqdm==4.48.1
+jsonschema>=3.0.0
```

Comment:
1. Oldest test requirements should be exact pinnings (not `>=`).
2. [requirements/requirements-test.txt](https://github.com/glotzerlab/signac-flow/blob/master/requirements/requirements-test.txt) should be pinned to the latest version, `jsonschema==4.0.1`. 4.0.1 was just released: https://pypi.org/project/jsonschema/